docs/_sources/python/generated/pyarrow.flight.Result.rst.txt

```diff
@@ -1,28 +1,6 @@
-pyarrow.flight.Result
+﻿pyarrow.flight.Result
 =====================
 
 .. currentmodule:: pyarrow.flight
 
-.. autoclass:: Result
-
-   
-   .. automethod:: __init__
-
-   
-   .. rubric:: Methods
-
-   .. autosummary::
-   
-      ~Result.__init__
-   
-   
-
-   
-   
-   .. rubric:: Attributes
-
-   .. autosummary::
-   
-      ~Result.body
-   
-   
\ No newline at end of file
+.. autoattribute:: Result
```

Comment:
This change looks a bit fishy. Checking the online docs, the flight docs indeed seem to be missing: https://arrow.apache.org/docs/python/api/flight.html